### PR TITLE
updated home screen

### PR DESCRIPTION
**Pull request by aryan-bhardwaj**
used new background for Home page

### CODE REVIEW

File: screens/Home.tsx [hunk 1]
```diff
@@ -1,154 +1,110 @@
...
   const [currNextEvent, setCurrNextEvent] = useState(null);
+  const [loading, setLoading] = useState(true); // To track loading state
 
   useEffect(() => {
-    const fetchCurrNext = async() => {
+    const fetchCurrNext = async () => {
       try {
         const event = await getCurrentOrNext(token);
         setCurrNextEvent(event);
+        console.log("NEXT/CURRENT EVENT", event);
```
Comment (divyack2) [inline screens/Home.tsx:27]: just remove the console logs and this should be good!

**Review by divyack2 — CHANGES_REQUESTED** [review]
lgtm! just remove the console logs/see if we can replace the console error and we can merge this :))

**Comment by divyack2** [comment]
also remove any commented code please!

File: screens/Home.tsx (hunk 1 above)
Comment (aryan-bhardwaj) [reply screens/Home.tsx:27]: done

**Review by divyack2 — APPROVED** [review]
lgtm!

**Pull request merged**